### ISSUE TITLE
function Beef.RoomPermission.__schema__/1 is undefined error thrown when creating a room

### ISSUE THREAD

**Issue by Choongkyu** [benawad/dogehouse#932]
**Describe the bug**
```
iex(1)> %UndefinedFunctionError{
  arity: 1,
  function: :__schema__,
  message: nil,
  module: Beef.RoomPermission,
  reason: nil
}

23:44:42.207 [error] function Beef.RoomPermission.__schema__/1 is undefined (module Beef.RoomPermission is not available)
error for op: get_current_room_users

23:44:42.208 [error]     (cowboy 2.8.0) /Users/choongkim/code/dogehouse/kousa/deps/cowboy/src/cowboy_websocket.erl:528: :cowboy_websocket.handler_call/6
    (cowboy 2.8.0) /Users/choongkim/code/dogehouse/kousa/deps/cowboy/src/cowboy_http.erl:254: :cowboy_http.loop/1
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
```

**To Reproduce**
1. Run full stack while on commit 74af4bd262778f3a53769bdea40425cba8f4c446 or even a commit a day before that one
2. try to create a room from frontend after logging in
3. See error

**Expected behavior**
Room creation without errors

**What device are you on?**
macbook pro

**Comment by ermalsh** [comment]
> **Describe the bug**
> A clear and concise description of what the bug is.
> 
> ```
> iex(1)> %UndefinedFunctionError{
>   arity: 1,
>   function: :__schema__,
>   message: nil,
>   module: Beef.RoomPermission,
>   reason: nil
> }
> 
> 23:44:42.207 [error] function Beef.RoomPermission.__schema__/1 is undefined (module Beef.RoomPermission is not available)
> error for op: get_current_room_users
> 
> 23:44:42.208 [error]     (cowboy 2.8.0) /Users/choongkim/code/dogehouse/kousa/deps/cowboy/src/cowboy_websocket.erl:528: :cowboy_websocket.handler_call/6
>     (cowboy 2.8.0) /Users/choongkim/code/dogehouse/kousa/deps/cowboy/src/cowboy_http.erl:254: :cowboy_http.loop/1
>     (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
> ```
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
> 1. Run full stack while on commit [74af4bd](https://github.com/benawad/dogehouse/commit/74af4bd262778f3a53769bdea40425cba8f4c446) or even a commit a day before that one
> 2. try to create a room from frontend after logging in
> 3. See error
> 
> **Expected behavior**
> Room creation without errors
> 
> **What device are you on?**
> macbook pro

Unable to reproduce on my end. If I were to guess, issue is ```Beef.RoomPermission``` should be ```Beef.Schema.RoomPermission```

**Comment by nadirabbas** [comment]
was fixed #851
